src/core.ts

```diff
@@ -266,7 +266,7 @@ function init() {
   resetUploads();
 
   if (config.customInstrumentation) {
-    let customInst = config.customInstrumentation();
+    let customInst = config.customInstrumentation(impressionId, cid, config.projectId);
```

Comment:
>let customInst = config.customInstrumentation(impressionId, cid, config.projectId); [](start = 4, length = 83)

Why this change?

Reply:
This is to allow webmasters to integrate Clarity into their own systems or analytics solution. E.g. a partner site can look up these values, store somewhere, and directly reference Clarity visualization. 

---
In reply to: [243067042](https://github.com/Microsoft/clarity-js/pull/116#discussion_r243067042) [](ancestors = 243067042)

Reply:
Moving conversation to a new comment in config.d.ts

---
In reply to: [245071226](https://github.com/Microsoft/clarity-js/pull/116#discussion_r245071226) [](ancestors = 245071226,243067042)